CI-CD/Pipeline-Samples/README.md

```diff
@@ -43,6 +43,24 @@ Finally, these samples depend on tasks from the [Dynamics 365 Finance and Operat
 - To setup the classic definition which can work with TFVC (Team Foundation Version Control) as well as Git repositories, import the **xpp-classic-ci.json** file from the pipelines landing page. (See: [Export and Import a Pipeline](https://docs.microsoft.com/en-us/azure/devops/pipelines/create-first-pipeline?view=azure-devops&tabs=java%2Cclassic%2Cbrowser%2Ctfs-2018-2#export-and-import-a-pipeline)). 
 - Alternatively you can run the **xpp-classic-ci.ps1** PowerShell script, provide [connection credentials including a PAT token](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=preview-page#create-personal-access-tokens-to-authenticate-access), and follow the script's prompts to create a new pipeline definition in your project. The token requires at least **Build: Read & execute** permissions, as well as **Agent Pools: Read**.
 
+## Classic Pipeline Variable Definitions
+
+| Variable | Default Value | Description |
+| ---	   | ---	   | ---	 |
+| `AppPackage`	| Microsoft.Dynamics.AX.Application.DevALM.BuildXpp	| Reference to the Application nuget package name	|
+| `AppSuitePackage`	| Microsoft.Dynamics.AX.ApplicationSuite.DevALM.BuildXpp	| Reference to the Applicaiton Suite nuget package name	|
+| `PlatPackage`	| Microsoft.Dynamics.AX.Platform.DevALM.BuildXpp	| Reference to the Platform nuget package name	|
+| `ToolsPackage`	| Microsoft.Dynamics.AX.Platform.CompilerPackage	| Reference to the compiler nuget package	|
+| `system.collectionId` | <> | The GUID of the TFS collection or Azure DevOps organization |

```

Comment:
Perhaps instead of putting the system ones here explicitly, refer to the docs link for Azure DevOps? Other than system.debug I'm not sure of common use cases to need these variables? Otherwise maybe add text that those are standard Azure DevOps ones that should only be changed if there's a specific reason to do so?
Just trying to keep it simple here, as well as not confuse standard features with anything to do with the X++ pipeline specifically...